### PR TITLE
Add support custom http headers in AzureMLOnlineEndpoint

### PR DESCRIPTION
**Pull request by yangmingzhe**
Description: 
support adding custom http headers when use AzureMLOnlineEndpoint through ContentFormatterBase

Issue: 
when using AzureMLOnlineEndpoint through ContentFormatterBase, I am unable to add any HTTP request header elements that I want.

Dependencies: 
no dependencies required

Details:
1. Add a dictionary-type member variable named 'headers' to class ContentFormatterBase
2. Pass the 'headers' of the instance object of "ContentFormatterBase" to the "client" member in class AzureMLOnlineEndpoint

Possible tag maintainer：
@baskaryan @hwchase17 @agola11 

Thanks!

### CODE REVIEW

**Comment by yangmingzhe** [comment]
@baskaryan, requesting review on this pull request.

File: libs/langchain/langchain/llms/azureml_endpoint.py [hunk 1]
```diff
@@ -71,11 +84,11 @@ def format_response_payload(self, output: str) -> str:
                     response_json = json.loads(output)
                     return response_json[0]["0"]
     """
-    content_type: Optional[str] = "application/json"
```
Comment (hwchase17) [inline libs/langchain/langchain/llms/azureml_endpoint.py:74]: is this change backwards comptabile? doesnt seem so

Comment (yangmingzhe) [reply libs/langchain/langchain/llms/azureml_endpoint.py:74]: yes, "content_type" and "accepts" have never been effective in this code, so can be removed

**Comment by yangmingzhe** [comment]
please review @hwchase17 , thanks

**Comment by yangmingzhe** [comment]
@hwchase17 please review again，thanks so much！

**Comment by yangmingzhe** [comment]
@hwchase17 please review again，thanks so much！

**Comment by yangmingzhe** [comment]
@hwchase17 
after carefully going through it, I think this pull request should be fine now. please kindly review it again. thanks for your help!

**Comment by yangmingzhe** [comment]
I'll submit a better implementation later on.

**Pull request closed without merging**